test/test_main.cpp

```diff
@@ -1009,6 +1009,22 @@ TEST_F(ConnectionParsing, AuthDigest) {
     ASSERT_TRUE(tx->request_auth_password == NULL);
 }
 
+TEST_F(ConnectionParsing, AuthBearer) {
+    int rc = test_run(home, "41-auth-bearer.t", cfg, &connp);
```

Comment:
Should rather be 100 than 41, right ?

Reply:
My bad. Amended.

Reply:
Looks good. Could you squash the commits together ?

Reply:
Done